### DISCUSSION TITLE
Autogen for image analysis with Claude3.5 API and MultimodalConversableAgent

### DISCUSSION THREAD

**Discussion by yananliusdu** [microsoft/autogen#4606] [General]
Is that possible in autogen to use MultimodalConversableAgent and Claude3.5 API for image analysis?  I tried following two methods that all failed:
1:  use such prompts  message= f"""What's on the image? <img 1.PNG>. """ same with using the openai api. this did not work
2: according to the examples in [https://docs.anthropic.com/en/docs/build-with-claude/vision#example-multiple-images](url), encode image into base64, which means convert image into str as an input for the api. However, it reported error 'TypeError: can only concatenate str (not "list") to str'

**Comment by rysweet** [comment]
Hi @yananliusdu - unfortunately without a more complete stack trace we can't really see which call is throwing the TypeError. Please post the whole stack trace and we might be able to help.
